### ISSUE TITLE
HDP model parameters K and T mismatch.

### ISSUE THREAD

**Issue by MaksimMMI** [RaRe-Technologies/gensim#2822]
#### Problem description

In documentation parameters K and T mismatch from original paper. Original paper states:
` We set the top-level truncation K = 150 and the second
level truncation T = 15. Here T<<K, since documents usually don’t have many topics.`

Documentation states: 
- K (int, optional) – Second level truncation
- T (int, optional) – Top level truncation   

#### Versions

```python
Darwin-18.7.0-x86_64-i386-64bit
Python 3.7.3 (default, Mar 27 2019, 16:54:48)
NumPy 1.16.1
SciPy 1.3.0
gensim 3.8.0
FAST_VERSION 1
```

**Comment by amirothman** [comment]
thank you for reporting. not sure i follow completely. can you elaborate a little bit more?

**Comment by piskvorky** [comment]
@MaksimMMI yeah sounds like a mix up. I haven't used this model myself – can you check which version is correct (paper or docs) and fix it? Thanks.

**Comment by ecoronado92** [comment]
@piskvorky It should be the other way around, in the original paper T is the second level truncation and K the first level truncation.

**Comment by piskvorky** [comment]
Well I hope at least the code is consistent. If it's indeed just a documentation bug, can you open PR to fix it?
